client/src/components/HomePage.tsx

```diff
@@ -11,13 +16,138 @@ const MainHeader = styled.h1`
   width: 100%;
 `;
 
-export type UserIdProps = {
-  userId: number;
-  setUserId: (userId: number) => void;
-};
+const SecondHeader = styled.h1`
+  font-size: 4rem;
+  color: ${CONTENT_COLOR};
+  text-align: center;
+`;
+
+const Content = styled.h2`
+  font-size: 3rem;
+  color: ${PRIMARY_COLOR};
+  text-align: center;
+`;
+
+const ContentText = styled.h3`
+  color: ${CONTENT_COLOR};
+  text-align: center;
+`;
+
+const ContentBlurb = styled.section`
+  border-bottom: 1px solid ${CONTENT_COLOR} ${QUARTER_OPAQUE};
+  border-top: 1px solid ${CONTENT_COLOR} ${QUARTER_OPAQUE};
+  border-right: 1px solid ${CONTENT_COLOR} ${QUARTER_OPAQUE};
+  border-left: 1px solid ${CONTENT_COLOR} ${QUARTER_OPAQUE};
+  flex-direction: row;
+  justify-content: space-between;
+  height: 220px;
+  width: 949px;
+  float: left;
+  background: ${PRIMARY_BACKGROUND_COLOR};
+`;
+
+const RedText = styled.h3`
+  color: ${"#A52A2A"};
```

Comment:
lol. if it's a hardcoded color, it could be just `color: #A52A2A`

but we should try to avoid hardcoding colors, as it will look weird if we switch themes. how does `SECONDARY_COLOR` look here?